### PR TITLE
First Model - Decision Tree

### LINKED ISSUE #30: Create decision tree

### LINKED ISSUE #31: Create criteria for amount of investment

### LINKED ISSUE #51: Enhance Decision Tree model with Hyperparameters and Specifications

### LINKED ISSUE #52: Integrate Model with recommendation engine
This should also close #31

### PR DESCRIPTION
**Pull request by guilhermefgs**
## Understand more about decision trees
[How does decision-tree work?](https://www.stat.berkeley.edu/~breiman/RandomForests/cc_home.htm)

Close #30 
Close #31 
Close #51
Close #52 

## Try this new feature:

```bash
$ cd tatu-re
$ pip install -r requirement.txt
$ cd tatu_re
$ python model.py
```

This code saves a trained model inside the **tatu_re/models** directory, you can load the model with the `load_model` function from models.py

The model is already integrated, so if you have a trained model saved, you can execute:
```bash
$ python main.py
```

### CODE REVIEW

**Comment by guilhermefgs** [comment]
**TO-DO**
- Define better hyperparameters [Planned not to do]
- Integrate into the recommendation engine [DONE]
  - needs to define criteria for the amount that should be invested [DONE - needs to be improved]
- Test set in another dataset [Planned as not to do in this PR

**Review by rodrigommorais — APPROVED** [review]
Só não sei se é melhor usar o preço de fechamento ou abertura, mas de resto tudo perfeito
